builder/simplify.cc

```diff
@@ -409,40 +473,48 @@ class simplifier : public node_mutator {
     void learn_from_equal(const expr& a, const expr& b) {
       if (const variable* v = a.as<variable>()) {
         // bounds of a are [b, b].
-        k.push_back(set_value_in_scope(bounds, v->sym, point(b)));
+        const std::optional<expr_info>& old_info = info_map[v->sym];
+        const alignment_type& al = old_info ? old_info->alignment :  alignment_type();
```

Comment:
It looks like this assigns a temporary to a reference? Maybe this should not be a reference?

Reply:
This const reference so should be fine? I changed to a non-reference anyway.